lib/request-handler.js

```diff
@@ -251,7 +251,7 @@ RequestHandler.prototype.prepareMultiple = function (queries, callbacksArray, op
   Error.captureStackTrace(this.stackContainer = {});
   var self = this;
   this.retryHandler = function () {
-    //Use a custom retryHandler
+    // Use a custom retryHandler
     self.prepareMultiple(queries, callbacksArray, options, callback);
```

Comment:
It should retry to prepare on all hosts selected by the LBP, not indefinitely.

We could add a test to make sure: use 2 hosts, warmup, pause both of them and try to send a prepared batch. The expected error should be instance of `NoHostAvailableError`.